common/src/main/java/com/wynntils/models/items/items/game/CraftedGearItem.java

```diff
@@ -156,6 +158,11 @@ public GearTier getGearTier() {
         return GearTier.CRAFTED;
     }
 
+    @Override
+    public boolean meetsActualRequirements() {
+        return false; // TODO
```

Comment:
Should this be fixed in this PR? I assume the calculation wouldn't work correctly if we hold a weapon with a skill bonus, but we cannot use it? Does crafted items "count" for skill points?

Reply:
Crafteds should be handled separately, i'll check for this function in a bit